platform/broadcom/one-image.mk

```diff
@@ -74,3 +74,19 @@ $(SONIC_ONE_IMAGE)_DOCKERS = $(SONIC_INSTALL_DOCKER_IMAGES)
 endif
 
 SONIC_INSTALLERS += $(SONIC_ONE_IMAGE)
+
+# DNX broadcom build
+SONIC_ONE_DNX_IMAGE = sonic-broadcom-dnx.bin
+$(SONIC_ONE_DNX_IMAGE)_MACHINE = broadcom
+$(SONIC_ONE_DNX_IMAGE)_IMAGE_TYPE = onie
+$(SONIC_ONE_DNX_IMAGE)_INSTALLS += $(BRCM_OPENNSL_KERNEL)
+$(SONIC_ONE_DNX_IMAGE)_INSTALLS += $(PDDF_PLATFORM_MODULE)
+$(SONIC_ONE_DNX_IMAGE)_INSTALLS += $(SYSTEMD_SONIC_GENERATOR)
```

Comment:
we should have something similar to this one. $(SONIC_ONE_DNX_IMAGE)_LAZY_INSTALLS

Reply:
Updated by adding the following, to make sure we install the respective module as per _MACHINE $(SONIC_ONE_IMAGE)_LAZY_BUILD_INSTALLS = $(BRCM_OPENNSL_KERNEL) $(BRCM_DNX_OPENNSL_KERNEL).
Please take a look.